src/stylesheets/components/Auth/_button-submit.scss

```diff
@@ -0,0 +1,15 @@
+.button-submit {
+  $self: &;
+
+  &__button--white {
+    @include neuzeit-font-semi-bold;
```

Comment:
what we need here is just set the `font-weight` to 800, right? 
how about just set an variable `$font-weight-bold: 800` and use it here? we don't need set the font-family here. kind of redundant.